coderd/database/dbpurge/dbpurge_test.go

```diff
@@ -205,8 +205,8 @@ func TestDeleteOldWorkspaceAgentLogs(t *testing.T) {
 			}
 			return !containsAgentLog(agentLogs, t.Name())
 		}, testutil.WaitShort, testutil.IntervalFast)
-		require.NoError(t, err)
-		require.NotContains(t, agentLogs, t.Name())
+		assert.NoError(t, err)
```

Comment:
drive-by change: I'm fighting [this flake](https://github.com/coder/coder/issues/13801).

Reply:
It does not address, but should give more insights into real values of `agentLogs` and `err`. I can revert it though.

Reply:
reverted